variables.json

```diff
@@ -6,6 +6,7 @@
   "MANIFOLD_EVENTS_SYNC_SCHEDULE_INTERVAL": "@daily",
   "TUP_ACCOUNT_NAME": "tupress",
   "TUPRESS_HARVEST_SCHEDULE_INTERVAL": "@daily",
+  "TUPRESS_HARVEST_SCHEDULE_INTERVAL": "0 8 * * *",
```

Comment:
Does this need to replace the one above?

Reply:
Yes I'l fix it.